### ISSUE TITLE
Dataset 

### ISSUE THREAD

**Issue by yiwei0730** [Plachtaa/seed-vc#93]
@Plachtaa  Can I ask about the training data for the SVC model? According to the content of the paper, is the training data is emilia+M4singer? or there have other music dataset? Approximately how many hours does it take?

**Comment by Plachtaa** [comment]
It was emilia plus some other singing datasets

**Comment by yiwei0730** [comment]
Can i know what dataset name it is? Or what is the approximate duration time of this data set?
